app/controllers/moirai/translation_files_controller.rb

```diff
@@ -37,11 +37,10 @@ def open_pr
     private
 
     def handle_update(translation)
-      translation_from_file = @file_handler.parse_file(translation_params[:file_path])
-      if translation_from_file[translation.key] == translation_params[:value] || translation_params[:value].blank?
+      if translation_params[:value].blank? || translation_same_as_in_file?(translation.key, translation_params[:value], translation.find_file_path)
```

Comment:
```suggestion
      if translation_params[:value].blank? || translation_same_as_in_file?(translation_params[:value], translation_params[:value], translation.file_path)
```

Reply:
No?
The file_path won't be set if we deprecate it. Here the file path should be determined from the `locale` and the `key` by using the KeyFinder.

Reply:
Ignore my previous comment. It makes much more sense to rename the `find_file_path` method to `file_path`